ci/scripts/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+#   Copyright © 2017 Teclib. All rights reserved.
+#
+# test.sh is part of FlyveMDMInventoryAgent
+#
+# FlyveMDMInventoryAgent is a subproject of Flyve MDM. Flyve MDM is a mobile
+# device management software.
+#
+# FlyveMDMInventoryAgent is free software: you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 3
+# of the License, or (at your option) any later version.
+#
+# FlyveMDMInventoryAgent is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# ------------------------------------------------------------------------------
+# @author    Hector Rondon
```

Comment:
Hi, @hectorerb 
Add also your email address

Reply:
Yes, I must update the header for the entire project